client/components/dashboard/dashboard-version-service.js

```diff
@@ -113,14 +113,21 @@ DashboardVersionService.prototype.verifyAndUpdateModel = function(dashboard) {
   // current.
   if (version == this.currentVersion) {
     if (!goog.isDef(dashboard.version)) {
+      goog.asserts.assert(!goog.isNull(version.version));
```

Comment:
Fixed up the version.version type to not allow nulls (it initializes to an empty string on empty anyways).